src/MyTested.WebApi/Utilities/RouteResolvers/InternalRouteResolver.cs

```diff
@@ -30,7 +31,20 @@ public static class InternalRouteResolver
         /// <returns>Resolved route information.</returns>
         public static ResolvedRouteInfo Resolve(HttpConfiguration config, HttpRequestMessage request)
         {
-            config.EnsureInitialized();
+            try
+            {
+                config.EnsureInitialized();
+            }
+            catch (InvalidOperationException ex)
+            {
+                if (ex.IsRouteConstraintRelatedException())
+                {
+                    throw new UnresolvedRouteConstraintsException(@"An error occurred while resolving your routes. If you are using custom route constraints, 
```

Comment:
Don't use `@`. Just leave the message on one long line. It is OK when it is just an error message.